src/screens/Add.js

```diff
@@ -0,0 +1,192 @@
+import React, {useRef, useState} from 'react';
+import {
+  SafeAreaView,
+  KeyboardAvoidingView,
+  View,
+  StyleSheet,
+} from 'react-native';
+
+import {action} from '@storybook/addon-actions';
+import {KeyboardAwareScrollView} from '@codler/react-native-keyboard-aware-scroll-view';
+import SelectDropdown from 'react-native-select-dropdown';
+
+import LanguageSwitcher from '../components/LanguageSwitcher/LanguageSwitcher';
+import ToolBar from '../components/ToolBar/ToolBar';
+import ToolButton from '../components/ToolButton/ToolButton';
+import BackIcon from '../components/ToolButton/assets/back.svg';
+import ModeIcon from '../components/ToolButton/assets/mode.svg';
+import {LANGUAGE_NAMES} from '../data/dataUtils';
+import SectionHeading from '../components/SectionHeading/SectionHeading';
+import Textarea from '../components/Textarea/Textarea';
+import Button from '../components/NextButton/NextButton';
+import {generateId} from '../utils';
+import DropdownArrowIcon from '../icons/select-dropdown-arrow.svg';
+
+export default ({categories, navigation, addNewPhrase, nativeLanguage}) => {
+  const [englishPhrase, setEnglishPhrase] = useState('');
+  const [malagasyPhrase, setMalagasyPhrase] = useState('');
+  const [selectedCategoryValue, setSelectedCategoryValue] = useState('');
+  const dropdownRef = useRef({});
+
+  const isButtonEnable =
```

Comment:
I am not sure about this but I think  button is enable (isButtonEnable = true) if all englishPhrase, malagasyPhrase and selectedCategory are not empty string. It's a small thing but it confuses me. May be you can explain if you think you meant this.
Other than that good job.

Reply:
isButtonEnable**d**